samples/msal-node-samples/auth-code/index.js

```diff
@@ -58,7 +58,21 @@ const getTokenAuthCode = function (scenarioConfig, clientApplication, port) {
             
         if (req.query) {
             // Check for the state parameter
-            if(req.query.state) authCodeUrlParameters.state = req.query.state;
+             /**
+             * MSAL Node supports the OAuth2.0 state parameter which is used to prevent CSRF attacks.
+             * The CryptoProvider class provided by MSAL exposes the createNewGuid() API that generates random GUID
+             * used to populate the state value if none is provided.
+             * 
+             * The generated state is then cached and passed as part of authCodeUrlParameters during authentication request.
+             * 
+             * For more information about state,
+             * visit https://datatracker.ietf.org/doc/html/rfc6819#section-3.6
+             */
+            if(req.query.state) {
+                authCodeUrlParameters.state = req.query.state;
+            } else {
+                authCodeUrlParameters.state = cryptoProvider.createNewGuid()
+            }
```

Comment:
```suggestion
authCodeUrlParameters.state = req.query.state ? req,query.state : cryptoProvider.createNewGuid();
```

Reply:
Can also be: `authCodeUrlParameters.state = req.query.state || cryptoProvider.createNewGuid();`